modules/tensor_mechanics/src/utils/RankTwoScalarTools.C

```diff
@@ -40,52 +39,48 @@ getQuantity(const RankTwoTensor & tensor,
       val = vonMisesStress(tensor);
       break;
     case 1:
-      ///For plastic strain tensor (ep), tr(ep) = 0 is considered
-      val = equivalentPlasticStrain(tensor);
-      break;
-    case 2:
```

Comment:
We can take care of this in a separate commit, but it would greatly help code readability to use the enum keys here (and in the solid_mechanics version).